Content.Shared/_CM14/Xenos/Gut/XenoGutDoAfterEvent.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Content.Shared.DoAfter;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared._CM14.Xenos.Gut;
+
+[Serializable, NetSerializable]
+public sealed partial class XenoGutDoAfterEvent : DoAfterEvent
+{
+    public override DoAfterEvent Clone()
+    {
+        return this;
+    }
+}
```

Comment:
In cases where `Clone()` just returns `this`, you can make the event inherit from `SimpleDoAfterEvent` instead.
```suggestion
public sealed partial class XenoGutDoAfterEvent : SimpleDoAfterEvent;
```